.github/workflows/push.yaml

```diff
@@ -1,172 +1,66 @@
-name: push-pipeline
+name: push
+
 on:
   push:
-    branches:
```

Comment:
@ybettan can you add these branches filters on push trigger?
We will need to create images for `main` and the release branches.

Reply:
Yes, I need to update the image tag as well in this case from `master` to release-branch name.

Reply:
Updated, I didn't add the `branch` filter but I have updated the image that is such way it will be `main` for `main` branch and `release-v3` for branch `release-v3` for example.